crates/biome_js_analyze/src/aria_analyzers/nursery/no_interactive_element_to_noninteractive_role.rs

```diff
@@ -0,0 +1,98 @@
+use crate::aria_services::Aria;
+use biome_analyze::{context::RuleContext, declare_rule, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::jsx_ext::AnyJsxElement;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Enforce that non-interactive ARIA roles are not assigned to interactive HTML elements.
+    ///
+    /// Interactive HTML elements indicate controls in the user interface.
+    /// Interactive elements include `<a href>`, `<button>`, `<input>`, `<select>`, `<textarea>`.
+    /// Non-interactive HTML elements and non-interactive ARIA roles indicate content and containers in the user interface.
+    /// Non-interactive elements include `<main>`, `<area>`, `<h1>` (,`<h2>`, etc), `<img>`, `<li>`, `<ul>` and `<ol>`.
+    ///
+    /// [WAI-ARIA roles] should not be used to convert an interactive element to a non-interactive element.
+    /// Non-interactive ARIA roles include `article`, `banner`, `complementary`, `img`, `listitem`, `main`, `region` and `tooltip`.
+    ///
+    /// Source: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-interactive-element-to-noninteractive-role.md
+    ///
+    /// [WAI-ARIA roles]: https://www.w3.org/TR/wai-aria-1.1/#usage_intro
```

Comment:
```suggestion
    /// [WAI-ARIA roles](https://www.w3.org/TR/wai-aria-1.1/#usage_intro)
```

This link wasn't renderd.